### PR TITLE
Added tech-stack, local development and contributing guideline to Rea…

### PR DESCRIPTION
**Pull request by KrownWealth**
…dme.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,45 @@
...
+ * [Typescript](https://www.typescriptlang.org/)
+ * [Tailwind CSS](https://tailwindcss.com/)
+ * [Saas](https://sass-lang.com/)
+ * [React Query](https://tanstack.com/query/v3/)
+ * [Contect API](https://react.dev/reference/react/useContext)
+ * [Farmer motion or Gsap for animation](https://www.framer.com/motion/)
+ * [Node JS](https://nodejs.org/en/docs)
+ * [Express JS](https://expressjs.com/en/guide/routing.html)
+ * [SQL](https://sql-academy.org/en)
+
```
Comment (abu-hasib) [inline README.md:27]: if you can add Prerequisite. It will be awesome

**Review by abu-hasib — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,10 +1,74 @@
...
+ * [Next JS](https://nextjs.org/ )
+ * [Typescript](https://www.typescriptlang.org/)
+ * [Tailwind CSS](https://tailwindcss.com/)
+ * [Saas](https://sass-lang.com/)
+ * [React Query](https://tanstack.com/query/v3/)
+ * [Contect API](https://react.dev/reference/react/useContext)
+ * [Farmer motion or Gsap for animation](https://www.framer.com/motion/)
+ * [Node JS](https://nodejs.org/en/docs)
+ * [Express JS](https://expressjs.com/en/guide/routing.html)
+ * [SQL](https://sql-academy.org/en)
```
Comment (sgarciahelguera) [inline README.md:55]: I think there should not be SQL in the front end (in the backend either)

Comment (KrownWealth) [reply README.md:55]: You're right, SQL is a kind of database stuff.

I will do the edit now.

**Comment by abu-hasib** [comment]
@KrownWealth This looks good to me. You can add the requested changes and merge.

**Comment by abu-hasib** [comment]
We can always add more information as the project grows.

**Review by abu-hasib — APPROVED** [review]

**Comment by abu-hasib** [comment]
#12

**Comment by sgarciahelguera** [comment]
closes: #12

File: README.md [hunk 3]
```diff
@@ -1,10 +1,34 @@
...
+
+This repository contains the Frontend development workflow for CubeSeed.
+
+## Tech Stack
+ * [Next JS](https://nextjs.org/ )
+ * [Typescript](https://www.typescriptlang.org/)
+ * [Tailwind CSS](https://tailwindcss.com/)
+ * [Saas](https://sass-lang.com/)
+ * [React Query](https://tanstack.com/query/v3/)
+ * [Contect API](https://react.dev/reference/react/useContext)
```
[the remark this answers is not in the document]
Comment (sgarciahelguera) [reply README.md:51]: Right , it is "Context API" not "Conect API"

**Comment by sgarciahelguera** [comment]
This is already approved,  I think you should just fix that typo and merge it.

**Comment by KrownWealth** [comment]
Context API fixed. Awaiting project maintained to merge the PRs.

**Pull request merged**